web/client/components/map/leaflet/plugins/WMTSLayer.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const Layers = require('../../../../utils/leaflet/Layers');
+const CoordinatesUtils = require('../../../../utils/CoordinatesUtils');
+const L = require('leaflet');
+const objectAssign = require('object-assign');
+const {isArray} = require('lodash');
+const SecurityUtils = require('../../../../utils/SecurityUtils');
+
+
+var WMTS = require('../../../../utils/leaflet/WMTS');
+
+
+L.tileLayer.wmts = function(urls, options) {
+    return new WMTS(urls, options);
+};
+
+function wmtsToLeafletOptions(options) {
+    // NOTE: can we use opacity to manage visibility?
+    return objectAssign( {}, options.defaultWmtsParams, {
+        service: "WMTS",
+        request: "GetTile",
+        tileMatrixPrefix: options.tileMatrixPrefix,
+        layer: options.name,
+        styles: options.style || "",
+        format: options.format || 'image/png',
+        tileMatrixSet: options.tileMatrixSet || CoordinatesUtils.normalizeSRS(options.srs || 'EPSG:3857', options.allowedSRS),
+        version: options.version || "1.0.0",
+        tileSize: options.tileSize || 256,
+        minx: options.minx || -25.6640625,
```

Comment:
This is not a minx, this is more originY, I would also use 20037508.3428 as the default value.